hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -277,6 +277,7 @@ public final class OzoneConsts {
   public static final String S3_GETSECRET_USER = "S3GetSecretUser";
   public static final String S3_SETSECRET_USER = "S3SetSecretUser";
   public static final String S3_REVOKESECRET_USER = "S3RevokeSecretUser";
+  public static final String S3_NAMESPACE_URL = "http://s3.amazonaws.com/doc/2006-03-01/";
```

Comment:
We can reuse the existing `S3Consts.S3_XML_NAMESPACE ` instead and remove this.

Reply:
Good point.  I missed it because the string is wrapped:

https://github.com/apache/ozone/blob/5c91b44ad1d6e9c61073c3718cd62929b40ba2c0/hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/S3Consts.java#L74-L75

Reply:
Yes, we can unwrap this since our line limit is 120 char now.